lib/twilio-ruby/security/request_validator.rb

```diff
@@ -31,7 +30,7 @@ def secure_compare(a, b)
 
         res = 0
         b.each_byte { |byte| res |= byte ^ l.shift }
-        res == 0
+        res.zero?
```

Comment:
🤔  is this because of a lint warning? `.zero?` seems to be used for data structures comparison, eg. if an array size  is zero - think similar to `.empty?`, or when float should be considered zero `2.2-2.2 = 0.0 != 0`. `.zero?` also throws an error when supplied with a non-numeric value instead of just saying false.


Also `res == 0` is negligible faster than `res.zero?`

```
➜  twilio-ruby git:(optional-twiml-parameters) ✗ irb
irb(main):001:0> require 'benchmark'
=> true
irb(main):002:0> loop = 10000000
=> 10000000
irb(main):003:0> Benchmark.bmbm() do |x|
irb(main):004:1* x.report("zero?") {for i in 1..loop; 0.zero?; end}
irb(main):005:1> x.report("==0") {for i in 1..loop; 0 == 0; end}
irb(main):006:1> end
Rehearsal -----------------------------------------
zero?   0.500000   0.000000   0.500000 (  0.492425)
==0     0.400000   0.000000   0.400000 (  0.405967)
-------------------------------- total: 0.900000sec

            user     system      total        real
zero?   0.500000   0.000000   0.500000 (  0.502405)
==0     0.410000   0.000000   0.410000 (  0.415262)
=> [#<Benchmark::Tms:0x007fd86a027a98 @label="zero?", @real=0.5024049979983829, @cstime=0.0, @cutime=0.0, @stime=0.0, @utime=0.5, @total=0.5>, #<Benchmark::Tms:0x007fd86a027a20 @label="==0", @real=0.4152617219951935, @cstime=0.0, @cutime=0.0, @stime=0.0, @utime=0.4099999999999999, @total=0.4099999999999999>]
irb(main):007:0>

```

Ultimately your choice. 

Why not `res.ten? or res.onepointfive?` dumb rule.